Source/WebCore/css/makeSelectorPseudoElementsMap.py

```diff
@@ -31,9 +31,14 @@
 def enumerablePseudoType(stringPseudoType):
     output = ['CSSSelector::PseudoElement::']
 
+    # Use proper WebKit casing for enum.
     webkitPrefix = '-webkit-'
     if (stringPseudoType.startswith(webkitPrefix)):
-        stringPseudoType = stringPseudoType[len(webkitPrefix):]
+        stringPseudoType = stringPseudoType.replace('-webkit-', '-WebKit-')
```

Comment:
Same here. I’m not sure we should keep a local variable `webkitPrefix` if we only use it once.